File: solo-leveling-player-model/src/main/java/com/sleepkqq/sololeveling/player/model/entity/player/enums/PlayerBalanceTransactionType.java
package com.sleepkqq.sololeveling.player.model.entity.player.enums;

import com.sleepkqq.sololeveling.jimmer.enums.LocalizableEnum;
import org.babyfish.jimmer.sql.EnumItem;

public enum PlayerBalanceTransactionType implements LocalizableEnum {
  @EnumItem(ordinal = 0)
  IN,

  @EnumItem(ordinal = 1)
  OUT
}

File: solo-leveling-player-model/src/main/kotlin/com/sleepkqq/sololeveling/player/model/entity/player/enums/PlayerBalanceTransactionType.kt
package com.sleepkqq.sololeveling.player.model.entity.player.enums

import org.babyfish.jimmer.sql.EnumItem

enum class PlayerBalanceTransactionType {
	@EnumItem(ordinal = 0)
	IN,

	@EnumItem(ordinal = 1)
	OUT
}
